python/seldon_core/batch_processor.py

```diff
@@ -195,6 +206,7 @@ def _start_request_worker(
     retries: int,
     batch_id: str,
     payload_type: str,
+    batch_interval: int,
```

Comment:
could it be float? we may want to wait for 1.5 second :)

Reply:
Yeah good point, will update